plugins/grpc/forwarder.go

```diff
@@ -345,20 +345,32 @@ func (g *GRPCForwarder) SchedulerEvent(ctx context.Context, infos, fwdMetadata m
 
 //Forward send room or player status to specified server
 func (g *GRPCForwarder) Forward(ctx context.Context, event string, infos, fwdMetadata map[string]interface{}) (status int32, message string, err error) {
+
+	// Add forwarder metadata (from maestro config) to request
+	// Client metadata should take priority
+	metadata := g.config.GetStringMap("metadata")
```

Comment:
can't we store this value as a field in GRPCForwarder instead of retrieving from viper.Config every time?

Reply:
up